.github/workflows/arm64.yml

```diff
@@ -67,6 +67,14 @@ jobs:
           ${TAS_SCRIPTS}/wait-for-alfresco-start.sh "http://localhost:8080/alfresco/" 180
           docker ps
           curl -v --user admin.pipeline@alfresco.com:admin http://localhost:8080/alfresco/api/discovery
+      - name: "Persist environment setup failure logs"
+        id: persist_environment_failure_logs
+        run: |
+          # Get containers state
+          docker ps -a
+          # Get Alfresco repo logs
+          docker ps -aqf "ancestor=quay.io/alfresco/alfresco-pipeline-all-amps-repo:${TEST_GIT_TAG:-latest}" | xargs -L 1 docker logs --tail 3000
+        if: failure()
```

Comment:
To be more precise, this condition should be adjusted so that this job only runs in case the build actually fails when trying to set up the environment, which may not always be the case as the condition is currently typed out.

You can take inspiration from [here](https://github.com/Alfresco/acs-packaging/blob/master/.github/workflows/ci.yml#L144) on how to type a more precise condition _(**NOTE**: also requires adding an id to the step we're interested in)_.